setup.cfg

```diff
@@ -0,0 +1,36 @@
+[metadata]
+name = qutip-tensorflow
+url = https://qutip.org
+description = TensorFlow backend for QuTiP
+long_description = file: README.md
+long_description_content_type = text/markdown; variant=GFM
+keywords = quantum, physics, dynamics, gpu, linear-algebra
+license = BSD 3-Clause License
+license_file = LICENSE.txt
+classifiers =
+    Development Status :: 2 - Pre-Alpha
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Topic :: Scientific/Engineering
+    Operating System :: MacOS
+    Operating System :: POSIX
+    Operating System :: Unix
+    Operating System :: Microsoft :: Windows
+author = Alexander Pitchford, Nathan Shammah, Shahnawaz Ahmed, Neill Lambert, Eric Giguere, Boxi Li, Jake Lishman, Simon Cross
+author_email = qutip-admin@googlegroups.com
+platforms = Linux, Mac OSX, Unix, Windows
+
+[options]
+package_dir =
+    =src
```

Comment:
One sees many Python packages with both folder layouts, and proponents of both. This folder structure matches that used by `qutip_qip` and is the one we're trying to standardize on for the family of QuTiP packages. Perhaps one day we will update `qutip` core to use this package structure too, but there is enough other stuff going on at the moment that we should probably leave things to settle down a bit first (e.g. finish the solvers, merge master into dev.major, etc).

Reply:
With this particular package, and possibly even more with `qutip-cupy`, I thought also thought that the `src/` structure was more appropriate, because there's a nonzero chance that the repository may produce more than one distributable package (like CuPy), and they might be better organised in their own directory.  I didn't know entirely how likely that was, but as long as it's something that _might_ happen, I thought it was neater to go this route - I don't see any huge disadvantage to it, and multiple-package discovery is very neat and safe like this.